core/src/components/action-sheet/action-sheet.md.vars.scss

```diff
@@ -70,10 +70,10 @@ $action-sheet-md-button-height:                         52px !default;
 $action-sheet-md-button-text-color:                     $text-color-step-150 !default;
 
 /// @prop - Font size of the action sheet button
-$action-sheet-md-button-font-size:                      16px !default;
+$action-sheet-md-button-font-size:                      1rem !default;
 
 /// @prop - Padding top of the action sheet button
-$action-sheet-md-button-padding-top:                    0 !default;
+$action-sheet-md-button-padding-top:                    12px !default;
```

Comment:
I added padding here because the buttons looked too cramped when the text scaled up. This should have no impact on buttons at the default text scale

| without padding | with padding |
| - | - |
| ![IMG_0012](https://github.com/ionic-team/ionic-framework/assets/2721089/7866816e-a569-4e5c-9c14-79b5217ffda5) | ![IMG_0011](https://github.com/ionic-team/ionic-framework/assets/2721089/60230598-97d3-406a-908b-43543d6fa0ae) |